src/transformers/quantizers/quantizer_torchao.py

```diff
@@ -162,7 +162,7 @@ def _process_model_after_weight_loading(self, model):
 
     @property
     def is_serializable(self):
-        return False
+        return True
```

Comment:
Let's create a new arg for `is_serializable` called `safe_serialization` and we set it to `None` by default. This way, we return a warning if the user have set `safe_serialization=True ` for this quantizer. Note that in `save_pretrained`, an error will be raise afterwards: 

```py
        quantization_serializable = (
            hf_quantizer is not None and isinstance(hf_quantizer, HfQuantizer) and hf_quantizer.is_serializable
        )

        if hf_quantizer is not None and not _hf_peft_config_loaded and not quantization_serializable:
            raise ValueError(
                f"The model is quantized with {hf_quantizer.quantization_config.quant_method} and is not serializable - check out the warnings from"
                " the logger on the traceback to understand the reason why the quantized model is not serializable."
            )
```

Note that this will require to update this method for all quantization methods. LMK if this sounds good to you.

Reply:
yeah sounds good, what does `safe_serialization=None` mean btw?

Reply:
I mean just set the default value of this optional args to `None`.

Reply:
OK, I was trying to ask how we can handle None, right now I'm just treating it the same way we treat `False`, please let me know if we have to handle it differently